notebooks/responsibleaidashboard/tour.ipynb

```diff
@@ -732,7 +731,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
remove notebook metadata changes, I think this folder could be rolled back, since the changes are not necessary